lib/Listerine/helpers.ex

```diff
@@ -42,4 +42,45 @@ defmodule Listerine.Helpers do
     |> Enum.filter(fn x -> x != "" end)
     |> Enum.map(&String.upcase/1)
   end
+
+  def set_bot_commands(channel) do
+    new_map =
+      case File.read("config.json") do
+        {:ok, ""} ->
+          %{"bot_commands" => channel.id}
+
+        {:ok, body} ->
+          map = Poison.decode!(body)
+          Map.put(map, "bot_commands", channel.id)
+
+        _ ->
+          %{"bot_commands" => channel.id}
+      end
+
+    File.write("config.json", Poison.encode!(new_map), [:binary])
+  end
+
+  def bot_commands?(message) do
+    id =
+      case File.read("config.json") do
+        {:ok, ""} -> nil
+        {:ok, body} -> Poison.decode!(body)["bot_commands"]
+        _ -> nil
+      end
+
+    cond do
+      id == message.channel.id ->
+        true
+
+      true ->
+        Channel.send_message(
+          Channel.get(id),
+          make_mention(message.author) <> " Esse commando tem de ser utilizado nesta sala!"
+        )
+
+        false
```

Comment:
Shoudn't this be a case instead of a cond do?

Reply:
it could be like this
```Elixir
case id == message.channel.id do
  true -> true
  false -> do_things()
           false
end
```
but that just looks like an if and if the classic if true then true, In a future refactoring I'll probably change this to not do any IO and so it'll all be reduced to `id == message.channel.id`

Reply:
Refactor this then. Clearly this function is doing more than one thing and it isn't even implied by the name.

Reply:
You're not going to like the next commit...